src/polyglot-notebooks-vscode/README.md

```diff
@@ -20,8 +20,8 @@ The following languages are supported by Polyglot Notebooks:
 | KQL ([Kusto Query Language](https://learn.microsoft.com/en-us/azure/data-explorer/kusto/query/))    |        ✅       |       
 | HTML                                        |        ⛔                 |     
 | [Mermaid](https://mermaid.js.org/intro/)    |        ⛔                 |        
-| Python (Preview)                            |        ✅                 |          
-| R (Preview)                                 |        ✅                 |          
+| [Python](docs/jupyter-in-polyglot-notebooks.md)   |        ✅           |
```

Comment:
These will need to be absolute URLs for them to work from VS Code Marketplace and the extension.

Reply:
Good point, I'll remove it for now, and once the docs are published, Ill add the absolute URL